external_parser/unit_tests/test_example_joiner.cc

```diff
@@ -0,0 +1,42 @@
+#define BOOST_TEST_DYN_LINK
+
+#include "example_joiner.h"
+#include "test_common.h"
+#include <boost/test/unit_test.hpp>
+
+BOOST_AUTO_TEST_CASE(example_joiner_test) {
+  auto vw = VW::initialize("--quiet --binary_parser --cb_explore_adf", nullptr,
+                           false, nullptr, nullptr);
+
+  example_joiner joiner(vw);
+
+  std::string input_files = get_test_files_location();
+  auto interaction_buffer = read_file(input_files + "/cb_v2.fb");
```

Comment:
maybe add a comment of which command is used to generate cb_v2, f-reward_v2 and cb_simple?

Reply:
Good point, added a readme file under `unit_tests/test_files` so that we know how each file is generated